src/test/kotlin/com/ampnet/reportserviceeth/service/JpaServiceTestBase.kt

```diff
@@ -44,6 +48,9 @@ abstract class JpaServiceTestBase : TestBase() {
     @Mock
     protected lateinit var userService: UserService
 
+    @Mock
```

Comment:
Is there a special purpose to mock service? No need to change it, just curious.

Reply:
Not really, it was easier to change it that way because prior to this it had grpc service mocked. Since we are already testing the happy path at controller level, as well as individual repository tests it was sufficient to do it like this.